prototyping/MockArticle.py

```diff
@@ -0,0 +1,8 @@
+class Article:
+
+    def __init__(self, number, content, keywords):
```

Comment:
Could you add a comment of what you expect these inputs to be too? I can figure it out from the code in Search.py, but it is not the most convenient way to figure it out. 

Also it will help you and @jaimehisao coordinate here if expectations are explicitly stated.